### PR TITLE
Revert "AA-300: Create legacy interfaces folder in ERC-4337 Solidity …

### PR DESCRIPTION
**Pull request by DemonBarber1020**
…sources …"

This reverts commit 6f02f5a28a20e804d0410b4b5b570dd4b076dcf9.

### CODE REVIEW

**Review by DemonBarber1020 — COMMENTED** [review]
??

**Review by DemonBarber1020 — COMMENTED** [review]
What's going on

File: reports/gas-checker.txt [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+║ simple - diff from previous    │     2 │               │          41492 │               10398 ║
 ╟────────────────────────────────┼───────┼───────────────┼────────────────┼─────────────────────╢
-║ simple                         │    10 │        449053 │                │                     ║
+║ simple                         │    10 │        449675 │                │                     ║
 ╟────────────────────────────────┼───────┼───────────────┼────────────────┼─────────────────────╢
-║ simple - diff from previous    │    11 │               │          41560 │               10527 ║
+║ simple - diff from previous    │    11 │               │          41663 │               10569 ║
 ╟────────────────────────────────┼───────┼───────────────┼────────────────┼─────────────────────╢
-║ simple paymaster               │     1 │         82245 │                │                     ║
+║ simple paymaster               │     1 │         82312 │                │                     ║
```
Comment (DemonBarber1020) [inline reports/gas-checker.txt]: [](url)

**Comment by drortirosh** [comment]
why?

**Pull request closed without merging**